README.md

```diff
@@ -25,9 +25,7 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 
 On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/fastquorum/results).
 
-| Tools                                                                                                              | Description                                                                                                                   |
-| ------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------- |
-| <p align="center"><img title="Fastquorum Workflow (Tools)" src="docs/images/fastquorum_subway.png" width=100%></p> | <p align="center"><img title="Fastquorum Workflow (Description)" src="docs/images/fastquorum_subway.desc.png" width=100%></p> |
```

Comment:
These are actually different images. The left hand side, "Tools", `fastquorum_subway.png` lists the tool names and the right hand side, "Description", `fastquorum_subway.desc.png` describes what those tools do at each step.

Reply:
Mmm, I don't see the reason to have both to be honest.

Reply:
It feels a little redundant because most of the tools have descriptive names, but I can appreciate the intention to clarify what each is doing. We can discuss further when Nils is back, but I don't see this as a needed change for the release.

Reply:
Fine, we can release without changing this